### PR TITLE
Grouped IOCTL codes in one place

### PR DESCRIPTION
**Pull request by lufog**
I saw this trick in the book "Pavel Yosufovich - Windows Kernel Programming".

### CODE REVIEW

**Comment by dretax** [comment]
I'm a big fan of stucturing the code more understandable. I believe the best approach would be to make a separate project folder, or something where we can store global header files. Please make a new one rather than storing them in the driver project.

File: Garhal/Garhal.vcxproj [hunk 1]
```diff
@@ -118,6 +118,7 @@
     <ClInclude Include="garhal.h" />
     <ClInclude Include="gstructs.h" />
     <ClInclude Include="hide.h" />
+    <ClInclude Include="ioctlcodes.h" />
```
Comment (dretax) [inline Garhal/Garhal.vcxproj:121]: Please place the global headers to a completely different project folder. Name It "GarhalGlobals"

**Review by dretax — CHANGES_REQUESTED** [review]

File: Garhal/Garhal.vcxproj (hunk 1 above)
Comment (dretax) [reply Garhal/Garhal.vcxproj:121]: @lufog ?

Comment (lufog) [reply Garhal/Garhal.vcxproj:121]: Sorry, at this moment, I don't have any time to work with this. You can merge pr and rename files as you wish or close this pr.

**Pull request closed without merging**